tests/unit/browser/webkit/test_history.py

```diff
@@ -412,5 +382,8 @@ def test_init(backend, qapp, tmpdir, monkeypatch, fake_save_manager):
         # before (so we need to test webengine before webkit)
         assert default_interface is None
 
-    assert fake_save_manager.add_saveable.called
+    # prevent interference with future tests
     objreg.delete('web-history')
+    hist.setParent(None)
+    if backend == usertypes.Backend.QtWebKit:
+        QWebHistoryInterface.setDefaultInterface(None)
```

Comment:
Setup/teardown for this test should probably be moved into a fixture so the teardown still happens in case the test fails.